styles/styles.css

```diff
@@ -138,6 +140,10 @@ a {
   font-size: 14px;
 }
 
+main {
```

Comment:
Is this really required ?

Reply:
No, was leftover from a version of the tutorial nav menu. I removed it